docs/src/introduction_to_catalyst/catalyst_for_new_julia_users.md

```diff
@@ -1,181 +1,229 @@
 ```@example ex1
 rand(100, 100)^3.5
 nothing # hide
 ```
 (This code creates a random 100x100 matrix, and takes it to the power of 3.5)
 
-This is useful to know when you e.g. declare, simulate, or plot, a Catalyst model. The first time you run a command there might be a slight delay. However, subsequent runs will execute much quicker. This holds even if you do minor adjustments before the second run (such as changing simulation initial conditions).
+This is useful to know when you e.g. declare, simulate, or plot, a Catalyst model. The first time you run a command there might be a slight delay. However, subsequent runs will be much quicker. This holds even if you make minor adjustments before the second run (such as changing simulation initial conditions).
 
-## Installing and activating packages
-Except for some base Julia packages (such as Pkg, the package manager) that are available by default, Julia packages must be installed locally before they can be used. Most packages are registered with Julia, and can be added through the `Pkg.add("DesiredPackage")` command (where `DesiredPackage` is the name of the package you wish to install). We can thus install Catalyst:
+## [Package management in Julia](@id catalyst_for_new_julia_users_packages)
+Due to its native package manager, and a registry of almost all packages of relevancy, package management in Julia is unusually easy. In principle, Catalyst can be both installed, and then imported into a session, using only:
 ```julia
 using Pkg
 Pkg.add("Catalyst")
+using Catalyst
 ```
+However, over the next three subsections we describe some additional considerations and useful commands. It is recommended to read and understand these at a relatively early stage. For a more detailed introduction to Julia package management, please read [the Pkg documentation](https://docs.julialang.org/en/v1/stdlib/Pkg/).
 
-Here, the command `using Pkg` is required to activate the Pkg package manager.
+### [Setting up a new Julia environment](@id catalyst_for_new_julia_users_packages_environments)
+Whenever you run Julia, it will run in a specific *environment*. You can specify any folder on your computer as a Julia environment. If you start Julia in that folder, that environment will be used (or, this will at least be the case for most modes of running Julia). If you start Julia in a folder not corresponding to an environment, your *default* Julia environment is used. While it is possible to not worry about environments (and always use the default environment), this can lead to long-term problems as more packages are installed.
```

Comment:
When I start the REPL in a folder it does not load the environment in that folder automatically; it loads the default global environment.

Reply:
Noted, will adapt.